src/app/(main)/[locale]/[...path]/page.tsx

```diff
@@ -85,6 +91,13 @@ async function Page({ params }: Props) {
     return Page404;
   }
 
+  let eventPostings: IEventPosting[] = [];
+
+  const { data } = await loadStudioQuery<{
+    eventPostingsArray: IEventPosting[];
+  }>(EVENT_POSTINGS_QUERY, { language: params.locale });
+  eventPostings = data?.eventPostingsArray ?? [];
+
```

Comment:
Her er jeg usikker på om dette burde skrives om og legges inn annerledes?

Reply:
Vet ikke helt hva du sikter til her, men jeg foretrekker dette iallefall. Evt trenger man kanskje ikke begge variablene? 

```suggestion
 const { data } = await loadStudioQuery<{
  eventPostingsArray: IEventPosting[];
}>(EVENT_POSTINGS_QUERY, { language: params.locale });

const eventPostings = data?.eventPostingsArray ?? [];

```